### PR TITLE
Added support for rule priorities in LALR

### PR DESCRIPTION
**Pull request by erezsh**

### CODE REVIEW

File: lark/lark.py [hunk 1]
```diff
@@ -272,12 +272,7 @@ def __init__(self, grammar, **options):
...
         if self.options.priority == 'auto':
-            if self.options.parser in ('earley', 'cyk', ):
-                self.options.priority = 'normal'
-            elif self.options.parser in ('lalr', ):
-                self.options.priority = None
-        elif self.options.priority in ('invert', 'normal'):
-            assert self.options.parser in ('earley', 'cyk'), "priorities are not supported for LALR at this time"
+            self.options.priority = 'normal'
 
         assert self.options.priority in ('auto', None, 'normal', 'invert'), 'invalid priority option specified: {}. options are auto, none, normal, invert.'.format(self.options.priority)
```
Comment (MegaIng) [inline lark/lark.py:277]: Detail: change `none` to `None` to prevent people from trying to specify `priority='none'`

File: tests/test_parser.py [hunk 2]
```diff
@@ -1827,14 +1826,13 @@ def test_earley_prioritization(self):
             """
 
             l = _Lark(grammar)
-            # l = Lark(grammar, parser='earley', lexer='standard')
             res = l.parse("a")
             self.assertEqual(res.children[0].data, 'a')
```
Comment (MegaIng) [inline tests/test_parser.py:1831]: Maybe add a not-as-trivial example:
```
start: a | b

a.2: "A"+
b.1: "A"+ "B"?
```

**Review by MegaIng — APPROVED** [review]
Not much to so. Very nice and concise!

**Comment by erezsh** [comment]
Nice suggestions. Anything else?

File: tests/test_parser.py [hunk 3]
```diff
@@ -1829,6 +1829,20 @@ def test_prioritization(self):
...
+            """
+
+            l = _Lark(grammar)
+            res = l.parse("AAAA")
+            self.assertEqual(res.children[0].data, 'a')
+
+            l = _Lark(grammar, priority="invert")
+            res = l.parse("AAAA")
+            self.assertEqual(res.children[0].data, 'b')
+
```
Comment (MegaIng) [inline tests/test_parser.py:1845]: I imagined here to also test that the input `AAB` gets correctly identified and doesn't cause problems. But invert is also nice to test.

**Review by MegaIng — APPROVED** [review]

**Comment by MegaIng** [comment]
Btw, did you read my message over at gitter?

**Review by MegaIng — APPROVED** [review]

**Pull request merged**